profile/README.md

```diff
@@ -1,32 +1,43 @@
-# Arcalot
-
-<img align="right" width="200px" style="padding-left: 2em;" alt="Arcalot logo showing a shield with the Arcalot inscription on a hill with the silhouette of a castle in the background" src="https://github.com/arcalot/.github/raw/main/branding/arcalot.png">
-
-The Arcalot community develops tools, plugins, and libraries that you can use either standalone as a library, and/or via a user interface or CLI. You can run the tools locally, remotely, or as part of a bigger system. Arcalot:
-
-* Helps you create workflows with **normalized input and output schemas**
-* Provides you with **assisted and automated root cause analysis** for the workflows you create as well as CI and other log systems
-* Provides **stable plugins for several workloads**
+# <a href="https://arcalot.io">Arcalot</a>
+<a href="https://arcalot.io"><img align="right" width="200px" style="padding-left: 2em;"
+ alt="Arcalot logo showing a shield with the Arcalot inscription on a hill with the 
+ silhouette of a castle in the background" 
+ src="https://github.com/arcalot/.github/raw/main/branding/arcalot.png"></a>
+
+The Arcalot community is dedicated to developing modular tools, plugins, and libraries
+with flexible implementations independently or as complete end-to-end solutions. We
+believe in enabling the automation and portability of complex tasks and in
+pre-validating actions to avoid costly re-runs due to late failures and incompatible
+data.
 
 <br clear="right"/>
 
-## Arcaflow
+## <a href="https://arcalot.io/arcaflow">Arcaflow</a>
 
-<img align="left" width="200px" style="padding-right: 2em;" alt="Arcaflow logo showing a waterfall and a river with 3 trees symbolizing the various plugins" src="https://github.com/arcalot/.github/raw/main/branding/arcaflow.png">
+<a href="https://arcalot.io/arcaflow"><img align="left" width="200px" style="padding-right: 2em;" alt="Arcaflow logo showing a waterfall and a river with 3 trees symbolizing the various plugins" src="https://github.com/arcalot/.github/raw/main/branding/arcaflow.png"></a>
 
-Arcaflow is a workflow engine consisting of three main components:
+Arcaflow is a workflow system consisting of three main components:
 
-* Core engine
-* UI
-* Plugins (including SDKs for Go and Python to write your own plugins)
+* A core engine, statically-compiled and platform-portable, written in Go
```

Comment:
Not convinced "platform-portable" conveys much here, after "statically compiled". It's written in Go. We build and test it for several platforms. You can build it for more, if you want. 😆

Reply:
The "statically-compiled" term will read fine and be understandable to a fairly technical reader, but may be a foreign concept to a less-technical reader. The "platform-portable" term is added to cover a broader audience.

Reply:
The odd thing is, to me as a fairly technical reader, statically-compiled means pretty much the _opposite_ of platform-portable -- you probably cannot run a Linux binary on Windows, you cannot run an Arm binary on X86, and so on, whereas dynamically compiled languages like Python and Java can "run anywhere"....

What I think we want to say is that the engine is provided as ready-to-run binaries available for "your platform", and that the code is platform-independent and can be easily built with a simple Go command.